picam_streamer:
  ros__parameters:

    configuration:
      cap_width: 1280
      cap_height: 720
      frame_id: "camera"
      camera_name: "raspicam"
      display_width: 640
      display_height: 480
      framerate: 90
      # 0=reverse, 2=normal
      flip_mode: 2 
      camera_calibration_file: "package://rastreator_camera/param/picam_calibration.yaml"
        

